nodejs/lancedb/indices.ts

```diff
@@ -214,6 +233,43 @@ export class Index {
   static fts() {
     return new Index(LanceDbIndex.fts());
   }
+
+  /**
+   *
+   * Create a hnswpq index
+   *
+   */
+  static hnswpq(options?: Partial<HnswPqOptions>) {
```

Comment:
To be consistent with other camel case names:
```suggestion
  static hnswPq(options?: Partial<HnswPqOptions>) {
```